### ISSUE TITLE
Vanilla shadows "Prefer in PvM" not working at Grotesque Guardians?

### ISSUE THREAD

**Issue by korvl38** [117HD/RLHD#421]
Hi, i hope I'm not misunderstanding here but according to the hover description of Vanilla Shadows, the setting "Prefer in PvM" should retain the circular shadow of the falling rocks which is helpful, but disable the cast shadow of the rocks? That doesn't seem to be working here, it's very difficult to see Jagex's shadow already with HD on but the cast shadows make it worse. Thank you
![2024-05-22_16-49-35](https://github.com/117HD/RLHD/assets/101873426/87cc03fd-d6e5-4b35-865b-19c52feaac3b)

**Comment by korvl38** [comment]
Thanks a lot! cant wait

**Comment by aHooder** [comment]
Fixed in https://github.com/117HD/RLHD/commit/f84b1f121a65fc0f9e47787bb40c70d4fa18f0eb, which should be in the next release. I've also changed the behaviour of 'Prefer in PvM' to only hiding HD shadows in specific cases, which currently only includes falling crystals at Olm, and falling rocks at Grotesque Guardians. I have a feeling people who want to disable shadows for bosses too might as well just disable shadows entirely. If it turns out people were actually relying on that behaviour though, I might add another setting for it.
